apps/tup-cms/src/taccsite_custom/tup_cms/static/tup_cms/css/for-tup-cms/components/c-tag.css

```diff
@@ -10,43 +10,54 @@
 
 /* Press Release */
 [class*="tag--pr"],
-.blog-category-press-release {
-  /* To overwrite core-cms django.cms.blog.app.css */
-  background: #723B0C !important;
+/* To overwrite core-cms django.cms.blog.app.css */
+.app-blog .categories a:is(
+  .blog-category-press-release
+) {
+  background: #723B0C;
 }
```

Comment:
**Explanation.** Rather than add `!important`[^1] for all instances (blog or not), add only the specificity required for blog tags to apply this color. (Non-blog instances no longer need `!important`.[^2]) Also, wrap the target class in `:is(…)` to reduce redundancy when adding multiple target classes (like selectors further down this stylesheet).

[^1]: Why not? Because `!important` forces all future overwrites to use `!important` i.e. can cause a chain reaction, and does **not** tell a developer what is being overridden nor what specificity (strength) an override is required. [Reference.](https://confluence.tacc.utexas.edu/display/UP/UI+-+CSS+Style+Guide+-+Specificity#UICSSStyleGuideSpecificity-!important)
[^2]: Because of the CSS cascade layer fix in `apps/tup-cms/src/taccsite_cms/templates/assets_custom.html`.